File: 1179.py
ci = 0
cp = 0
i = [1,1,1,1,1]
p = [0,0,0,0,0]

for x in range(15):
	v = int(input())
	if v % 2 == 0:
		p[cp] = v
		cp += 1
		if cp == 5:
			for b in range(cp):
				print('par[%d] = %d' %( b , p[b]))
			cp = 0
	else:
		i[ci] = v
		ci += 1
		if ci == 5:
			for a in range(ci):
				print('impar[%d] = %d' %(a , i[a]))
			ci = 0

for c in range(ci):
	print('impar[%d] = %d' %(c , i[c]))

for b in range(cp):
	print('par[%d] = %d' %( b , p[b]))

File: c & c++/1179.c
#include <stdio.h>
 
int main() {
	int par[5]; 
	int impar[5];
	
	int ci = 0;
	int cp = 0;
	int x = 0;
	int z = 0;
	int v;
	
	for(z =0; z < 15; z++){
		scanf("%d", &v);
		
		if(v % 2 == 1 || v % 2 == -1){
			if(ci == 5){
				for(x = 0; x < 5;x++){
					printf("impar[%d] = %d\n", x,  impar[x]);	
				}
				for(x = 0; x < 5; x++){
					impar[x] = 0;	
				}
				ci = 0;
				
				impar[ci] = v;
				
				ci++;
			}else{
				impar[ci] = v;
				
				ci++;
			}
			
		}else{
			if(cp == 5){
				for(x = 0; x < 5;x++){
					printf("par[%d] = %d\n", x,  par[x]);	
				}
				for(x = 0; x < 5; x++){
					par[x] = 0;	
				}
				cp = 0;
				
				par[cp] = v;
				
				cp++;
			}else{
				par[cp] = v;
				cp++;
			}
		}
	}
	
	for(z = 0; z < ci; z++){
		printf("impar[%d] = %d\n", z,  impar[z]);	
	}
	for(z = 0; z < cp; z++){
		printf("par[%d] = %d\n", z,  par[z]);
	}
	
    return 0;
}